src/js/core/widget/core/Slider.js

```diff
@@ -80,7 +80,7 @@
 				events = ns.event,
 				/**
 			 	 * Widget options
-				 * @property {boolean} [options.type="normal"] Slider type. 'normal', 'center' or 'circle'
+				 * @property {boolean} [options.type="normal"] Slider type. 'normal', 'level-bar'
 				 * @property {string} [options.orientation="horizontal"] Slider orientation. horizontal or vertical
```

Comment:
if we make orinetation not supported, shouldn't we remove it from API description?

Reply:
We can remove these options but widget will stop display alert about deprecated options.
I can remove it, what you think?

Reply:
Done.